packages/create-webpack-app/src/generators/init/vue.ts

```diff
@@ -0,0 +1,249 @@
+import { NodePlopAPI, Answers, ActionType } from "../../types";
+import { dirname, resolve, join } from "path";
+import ejs from "ejs";
+import { DynamicActionsFunction } from "node-plop";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+export default async function (plop: NodePlopAPI) {
+  // dependencies to be installed
+  const dependencies: Array<string> = [
```

Comment:
I have done this in a recent commit to all the generators and reverted the change here so that conflict doesnt arise.
see this commit: [e84b1af](https://github.com/webpack/webpack-cli/pull/4242/commits/e84b1afae3502e6df1903157a1fe495ba43a986d)